components/ht16k33_alpha/font.h

```diff
@@ -1,6 +1,6 @@
 // based on Adafruit backpack library
 
-static const uint16_t alphafonttable[] PROGMEM =  {
+static const uint16_t alphafonttable[] PROGMEM = {
```

Comment:
It might be nice to add some "#define"s to only include the character table if the corresponding display type is actually being used; in order to not waste ROM space.
Does it make sense?

Reply:
After the split in different components, this comment is obsolete